overrides/scripts/item_descriptions.zs

```diff
@@ -3,38 +3,89 @@ import crafttweaker.api.item.IItemStack;
 import crafttweaker.api.text.Component;
 import mods.jeitweaker.ingredient.JeiIngredient;
 #priority -11
+
+
 function i(m as IItemStack, s as string) as void {
```

Comment:
best practice would rename this function to be more clear. you might be able to right click on it and "rename symbol" to rename all. if not don't worry about it